lib/locales/el.ts

```diff
@@ -0,0 +1,493 @@
+    louise_de_marillac_religious: 'Αγίας Λουίζας de Marillac',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Μάρτιος, p33
+    lucy_of_syracuse_virgin: 'Αγίας Λουκίας, παρθενομάρτυρος',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Δεκέμβριος, p42
+    luke_evangelist: 'Αγίου Λούκα, Ευαγγελιστή',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Οκτώβριος, p40
+    lydia_of_philippi: 'Αγίας Λυδίας της Φιλιππησίας',  // based on: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Αύγουστος, p38
+    lydia_of_philippi_first_european_christian: 'Αγίας Λυδίας της Φιλιππησίας, πρώτης Χριστιανής της Ευρώπης',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Αύγουστος, p38
+    mammes_of_caesarea_martyr: 'Αγίου Μάμα, μάρτυρος',
+    marcellin_champagnat_priest: 'Αγίου Μαρκελλίνου Champagnat, πρεσβυτέρου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιούνιος, p36
+    marcellinus_of_ancona_bishop: 'Αγίου Μαρκελλίνου, επισκόπου',
+    marcellinus_of_rome_and_peter_the_exorcist_martyrs: 'Αγίων Μαρκελλίνου και Πέτρου, μαρτύρων',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιούνιος, p36
+    margaret_mary_alacoque_virgin: 'Αγίας Μαργαρίτας Μαρίας Alacoque, παρθένου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Οκτώβριος, p40
+    margaret_of_antioch_virgin: 'Αγίας Μαρίνας, παρθενομάρτυρος',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιούλιος, p37
+    margaret_of_scotland: 'Αγίας Μαργαρίτας της Σκοτίας',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Νοέμβριος, p41
+    maria_goretti_virgin: 'Αγίας Μαρίας Goretti, παρθενομάρτυρος',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιούλιος, p37
+    mark_evangelist: 'Αγίου Μάρκου, Ευαγγελιστή',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Απρίλιος, p34
+    martha_of_bethany_mary_of_bethany_and_lazarus_of_bethany: 'Αγίων Μάρθας, Μαρίας και Λαζάρου',  // based on: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιούλιος, p37
+    martin_de_porres_religious: 'Αγίου Μαρτίνου de Porres, μοναχού',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Νοέμβριος, p41
+    martin_i_pope: 'Αγίου Μαρτίνου Αʹ, πάπα και μάρτυρος',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Απρίλιος, p34
+    martin_of_tours_bishop: 'Αγίου Μαρτίνου, επισκόπου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Νοέμβριος, p41
+    mary_magdalene: 'Αγίας Μαρίας της Μαγδαληνής',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιούλιος, p37
+    mary_magdalene_de_pazzi_virgin: 'Αγίας Μαρίας Μαγδαληνής de’ Pazzi, παρθένου',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Μάϊος, p35
+    mary_mother_of_god: 'Ογδοάδα των Χριστουγέννων: Της Υπεραγίας Θεοτόκου Μαρίας',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Ιανουάριος, p31
+    mary_mother_of_the_church: 'Tης Υπεραγίας Θεοτόκου Μαρίας, μητέρας της Εκκλησίας',  // src: mr_el_2006_ed3, Γενικό Ρωμαϊκό εορτολόγιο, Μάϊος, p35
```

Comment:
According to the [_Liturgical Calendar of the Archdiocese of Athens](), p79, the new name of the celebration is `Της Υπεραγίας Θεοτόκου και Αειπαρθένου Μαρίας, Μητέρας της Εκκλησίας`.